epicweb-dev/epic-stack
# Andrecasal/issue94

Fixes #94

Install Playwright browsers on setup

## Test Plan

1. Run with local template: `npx create-remix@latest --typescript --install --template ./epic-stack`
2. Check the Playwright browsers where installed ✅

## Checklist

Do we need some tests/docs for this?

## Screenshots

N/A
